src/UI/Main.java

```diff
@@ -0,0 +1,45 @@
+package UI;
+
+/**
+ * @Author	 : Shiva Dhanush
+ * This file brings up the initial startSession page.
+ */
+
+import javafx.application.Application;
+import infrastructure.validation.logger.*;
+import javafx.fxml.FXMLLoader;
+import javafx.stage.Stage;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.layout.BorderPane;
+import org.json.*;
+
+import infrastructure.content.*;
+
+public class Main extends Application {
+	static ILogger logger = LoggerFactory.getLoggerInstance();
+	/**
+	 * This function opens the start session page.
+	 * @param PrimaryStage This is Stage for showing fxml page.
+	 * @returns nothing
+	 */
+	@Override
+	public void start(Stage primaryStage) {
+		try {
+			/**
+             * Loading the startSession page
+             */
+			Parent root = FXMLLoader.load(getClass().getResource("StartSession.fxml"));
+			Scene scene = new Scene(root,600,800);
+			primaryStage.setScene(scene);
+			primaryStage.show();
+	    	logger.log(ModuleID.UI, LogLevel.SUCCESS, "Startsession page has been opened.");
```

Comment:
```suggestion
	    	logger.log(
	    	    ModuleID.UI, 
	    	    LogLevel.SUCCESS, 
	    	    "Startsession page has been opened."
	    	 );
	    	
```

Also, fix the indendation